src/Darryldecode/Cart/Cart.php

```diff
@@ -784,4 +755,14 @@ public function setThousandsSep($thousands_sep)
     {
         $this->thousands_sep = $thousands_sep;
     }
+
+    /**
+     * @param $name
+     * @param $value
+     * @return mixed
+     */
+    protected function fireEvent($name, $value = [])
+    {
+        return $this->events->fire($this->getInstanceName() . '.' . $name, array_values([$value, $this]), true);
```

Comment:
To clarify
`return $this->events->fire($this->getInstanceName() . '.' . $name, array_values([$value, $this]));`